e2e/accounts/accounts_rpc_test.go

```diff
@@ -43,14 +43,13 @@ func (s *AccountsTestSuite) TestRPCEthAccountsWithUpstream() {
 	// FIXME(tiabc): Stop skipping after https://github.com/status-im/status-go/issues/424
 	s.T().Skip()
 
-	s.StartTestBackend(
-		params.RopstenNetworkID,
-		e2e.WithUpstream("https://ropsten.infura.io/z6GCTmjdP3FETEJmMBI4"),
-	)
+	addr, err := GetNetworkURLFromID(params.RopstenNetworkID)
```

Comment:
1. Rename to the valid name: `GetRemoteURLFromID`
2. Why Ropsen? It may be any network specified in `-network`

Reply:
Currently we default to `statuschain` as network id to be used when no network is provided, but `statuschain` has no remove URl, hence why this tests usage of `Ropsten` remains.

Reply:
Maybe, we shall use what's provided in `-network` and just skip this test for statuschain?